devices/org.osbuild.loopback

```diff
@@ -106,7 +108,8 @@ class LoopbackService(devices.DeviceService):
         dir_fd = -1
         try:
             dir_fd = os.open(devpath, os.O_CLOEXEC | os.O_PATH)
-            self.lo.mknod(dir_fd)
+            with ctx.suppress_oserror(errno.EEXIST):
+                self.lo.mknod(dir_fd)
```

Comment:
Is that because devtmpfs created it, or because it is one of those pre-created static devices? I mean, we just allocated a new loop-devices, so it should not exist.

Reply:
@lavocatt is experimenting using the existing osbuild code for device handling and mounting in `image-info`, and there we (currently) are operating on the host.

Reply:
I was more wondering which code creates it on the host? Is this created by devtmpfs? Or why do they exist on the host?

Reply:
I assume left-over from previous runs, maybe? I have `/dev/loop{1,2,3}` on relatively fresh booted laptop. `losetup -f` yields `loop0`.

Reply:
Right. Those are the pre-created once from pre-loop-setup days. You can disable them with a module-parameter, if one would care. Anyway, yeah, those will pre-exist. I thought that lo-new would skip those, but apparently it doesn't. Meh.

Change looks good!